.github/workflows/msbuild.yml

```diff
@@ -0,0 +1,44 @@
+name: MSBuild
+
+on: [push]
+
+env:
+  # Path to the solution file relative to the root of the project.
+  SOLUTION_FILE_PATH: .
```

Comment:
Does this not have to point to the .sln?

Reply:
I don't know, it works perfectly like that and that was automatically generated from github xD

Reply:
Looks like it just builds everything in current directory when you give it "." (it's just how msbuild handles arguments)